src/configure/configurers/azurePipelineConfigurer.ts

```diff
@@ -42,16 +42,6 @@ export class AzurePipelineConfigurer implements Configurer {
 
     public async getInputs(inputs: WizardInputs): Promise<void> {
         try {
-            if (inputs.sourceRepository.repositoryProvider === RepositoryProvider.Github) {
-                inputs.githubPATToken = await this.controlProvider.showInputBox(constants.GitHubPat, {
-                    placeHolder: Messages.enterGitHubPat,
-                    prompt: Messages.githubPatTokenHelpMessage,
```

Comment:
One more reason to not touch this code is that it uses a different prompt message than the github workflow flow one.